src/pseudopeople/utilities.py

```diff
@@ -192,6 +192,35 @@ def cleanse_integer_columns(column: pd.Series) -> pd.Series:
     return column
 
 
+def number_of_tokens_per_string(s1, s2):
+    """
+    Calculates the number of tokens in each string of a series.
+    s1 is a pd.Series of tokens and we want to see how many tokens exist in each
+    string of s2.
+    """
+
+    s2 = s2.astype(str)
+    strings = s2.unique()
+    tokens_per_string = pd.Series(
+        (sum(occurrences(s, str(token)) for token in s1) for s in strings), index=strings
+    )
+
+    number_of_tokens = s2.map(tokens_per_string)
+    number_of_tokens.index = s2
+    return number_of_tokens
+
+
+# https://stackoverflow.com/a/2970542/
+def occurrences(string, sub):
```

Comment:
nit: we tend to name functions as an action, e.g. `count_occurences`.

This really threw me off above on line 205 when you used it b/c it didn't look like a user-defined function and I thought there was a native python thing I didn't know about